circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
+  python:
+    version: 2.7.12
+  environment:
+    # Set matplotlb backend to the non-interactive antigrain image lib
+    MPLBACKEND: Agg
+dependencies:
+  pre:
+    # 'models' folder is cached, don't download twice
+    - cd models && wget -nc http://www.tmbdev.net/en-default.pyrnn.gz
+    # Pipe to cat to hide the progress bars
```

Comment:
`-q` hides INFO messages, including build errors which I do want to see. There is no option like `--no-progress-bar` to hide just the animated progress bar which messes with CircleCI web rendering of the console. Piping to cat hides the progress bar and keeps the error output.

BTW: I'm aware of #37 (thanks!) and reviewing your commits. Ideally, we'll have CircleCI for virtualenv and TravisCI for conda testing.